src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -158,4 +158,22 @@ public function getFlashes(string|array $types = null): array
 
         return $result;
     }
+
+    public function getCurrent_Route(): ?string
+    {
+        if (!isset($this->requestStack)) {
+            throw new \RuntimeException('The "app.current_route" variable is not available.');
+        }
+
+        return $this->getRequest()?->attributes->get('_route');
+    }
+
+    public function getCurrent_Route_Parameters(): ?array
+    {
+        if (!isset($this->requestStack)) {
+            throw new \RuntimeException('The "app.current_route_parameters" variable is not available.');
+        }
+
+        return $this->getRequest()?->attributes->get('_route_params');
```

Comment:
should we actually return `null` when we don't have a request, or should we return an empty array ? Returning an empty array would simplify the signature to me.

Reply:
Good idea, updated, thanks!